src/Task/Infection.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Infection task
+ */
+class Infection extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'infection';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+
+        $resolver->setDefaults([
+            'threads' => null,
+            'test_framework' => null,
+            'only_covered' => null,
+            'configuration' => null,
+            'min_msi' => null,
+            'min_covered_msi' => null,
+            'mutators' => null,
+            'triggered_by' => ['php'],
+        ]);
+
+        $resolver->addAllowedTypes('threads', ['null', 'int']);
+        $resolver->addAllowedTypes('test_framework', ['null', 'string']);
+        $resolver->addAllowedTypes('only_covered', ['null', 'bool']);
```

Comment:
Since it's a bool, doesn't it make more sense to set the default to false and only support bool?